\documentclass[12pt]{article}
\usepackage{geometry} 
\usepackage{indentfirst}
\usepackage{hyperref}
\usepackage{color}
\usepackage{comment}
\usepackage[pdftex]{graphicx}  
\usepackage{caption}
\usepackage{natbib}
\usepackage{mathtools}
\usepackage{units}
\usepackage{booktabs}
\usepackage{authblk}
\renewcommand{\baselinestretch}{1.5}
\geometry{a4paper} 
\bibliographystyle{apalike}

%----------------------------------------
%AUTHORS
%----------------------------------------
\title{... evidence for selection on maize centromere repeats}
\author[1]{Christopher J. Fiscus}
\author[1]{Siddharth Bhadra-Lobo}
\author[1]{Paul Bilinski} 
\author[1]{Tyler V. Kent}
\author[1]{Nicole Soltis}
\author[1]{Timothy T. Yang} 
\author[1,2]{Jeffrey Ross-Ibarra\thanks{rossibarra@ucdavis.edu}}

\affil[1]{Department of Plant Sciences, University of California Davis}
\affil[2]{Center for Population Biology and Genome Center, University of California Davis}

\date{}

%-----------------------------------------------------------------------------------------------------------------
% BEGIN DOCUMENT
%-----------------------------------------------------------------------------------------------------------------
\begin{document}
\maketitle

\begin{abstract}
Abstract goes here.
\end{abstract}

\begin{footnotesize}
\noindent{\bf{\underline{GWAS}} results are available at \url{} }
\end{footnotesize}

\section*{Notes}

Will delete this section later.  Some simple tips:
Bold can be done {\bf like this}.  Italics are \emph{easy} too.  Math formulas are also easy: $Area=\pi r^2$.
Here's how you enter a citation \cite{Wolfgruber2009}.
Here's another way to enter a citation \citep{Wolfgruber2009}.
All citations need to have a bibtex formatted entry in the file "references.bib" in this same repo. You can add comments to the file using "\%" which will not be shown in the final pdf.  

\section{Introduction}

Centromeres are dense, heterochromatic regions on chromosomes that act as an assembly site for the kinetochore complex during meiosis and mitosis and link sister chromatids together.  The role of the centromere is almost universally conserved among eukaryotes, despite a lack of sequence similarity in the centromeres of closely related species \cite{Henikoff2001}.  Centromeric DNA is some of the most rapidly evolving sequence in eukaryotes \cite{Csink1998}, \cite{Henikoff2001} and is hypothesized to play a significant role in speciation \cite{Henikoff2001}.  The functional centromere is epigenetically defined by the presence of nucleosomes featuring a histone H3 variant unique to the centromere, called CenH3 \cite{Henikoff2001}.  CenH3 replaces the standard histone H3 protein in the H2A-H2B-H3-H4 histone core and is thought to be responsible for the centromere's exclusive heterochromatic state. 

In maize, the DNA sequence of the functional centromere is composed primarily of CentC satellites and retrotransposons \cite{Nagaki2003}.  Each maize centromere has a unique organization of these elements \cite{Ananiev1998}, yet functions in the same capacity.  CentC satellite tandem repeats are generally 156 base pairs (bp) in length, are localized in centromere regions, and are unique to maize \cite{Ananiev1998}.  The sequence of CentC is largely conserved between individuals but single nucleotide polymorphisms have been observed \cite{Ananiev1998}.  

% Expand description of Centromere
% Add to why CentC
% Centromere Drive

Here, we present evidence for/against drive on centromere repeat abundance in maize.     

\section{Materials and Methods}

\subsection{Estimation of CentC Abundance} %Nicole

\subsection{CentC Mapping on B73 Reference Genomes, Versions 1-3} %Sid 

79 nucleotide sequences of Maize Centromeric repeats CentC were downloaded from \cite{ncbi nucleotide database}. Using BLAST to align the 79 queries against the B73 reference genome versions 1, 2, and 3 \cite{MaizeGDB} \cite{BLAST}. Reference Genome version 3 was found downloaded from  \url{http://plants.ensembl.org/Zea_mays/Info/Index/}. The query file was built from all 79 CentC repeats concatenated into one header less file. The BLAST outputs were formatted into a tab delimited table and read into {\bf RStudio} and graphed with {\bf R} \cite{R} \cite{RStudio}. The BLAST hittable for the 79 CentC hits from each individual reference Genome version was mapped to each of the 10 Maize chromosomes by finding the length of each chromosome per reference genome version. Chromosome lengths were found taking a character count, excluding header lines, of each chromosome from the reference genome .fasta format files. 

%Cite ncbi nucleotide database
%Cite B73 MaizeGDB
%Cite refines versions 1 and 2 from Ensemble

\subsection{Relative Genome Size Estimation } %Sid

In order to relate CentC abundance to Centromere size, the size of the genome being referenced against must also be known, relative to the other genomes in which CentC abundance is also being estimated \cite{ncbi nucleotide database}.. To estimate genome size in the Jiao inbred maize lines, sequenced complementary DNA (cDNA) from \emph{Zea mays} reference genome version 3 was aligned against the Jiao dataset \cite{Jiao2012}. Using the Burroughs-Wheeler Aligner (BWA) mem function, number of Reads Mapped and Total Reads from each line was found \cite{BWA}. SAMtools was used to read the BWA mem alignment values from their default Binary Alignment Map (bam) format \cite{SAMtools}. These values were read into {\bf RStudio} as a tab delimited table and {\bf R} was used to aggregate the data based on the line it came from to avoid duplicate lines \cite{R} \cite{RStudio}. From the consolidated data, simple column arithmetic was done to divide Reads Mapped by Total Reads then multiplied by 100 to find percent abundance CentC for the Jiao dataset. 

%Cite Jiao Lines.
%Cite CentC database


Finding abundance of CentC in Megabases (Mb)

$\dfrac{CentC}{Total Reads}\times Mb$ = Mb CentC	

Finding genome size with relation to abundance of cDNA in Megabases (Mb)

$\dfrac{\dfrac{1}{cDNA}}{Total Reads}$ $\alpha$ Mb




\subsection{Genome-Wide Association Analyses} %Chris
Population structure was inferred computationally for a randomly generated 10,000 single nucleotide polymorphism (SNP) subset from Jiao et al dataset \cite{Jiao2012} using \emph{structure} software version 2.3.4 accessed from \url{pritchardlab.stanford.edu/structure.html}.  The basic algorithm employed in this software package was first described by Pritchard, Stephens, \& Donnelly \cite{Pritchard2000} and extensions to the method were reported by Falush, Stephens \& Pritchard \cite{Falush2003}, \cite{Falush2007} and Hubisz, Falush, Stephens \& Pritchard \cite{Hubisz2009}.  The software was used to generate Q matrices and to estimate the natural log probability of the data for 1-20 maximum populations, represented by \emph{K}.  The length of burn-in used in the analysis was 10000 and the number of MCMC reps after the burn-in period was 20000.  Candidate \emph{K} values were determined using the $\Delta$ K method and the resulting Q matrices, inferring relatedness, were used in further analyzes.    

% more info on burnin and MCMC reps and "natural log probability of data"
% method to determine K needs citation (Ln P(D) & delta K)
% add section about GWAS using TASSEL 

Genome-wide association analyzes (GWAS) of the Jiao et. al dataset \cite{Jiao2012} were completed using TASSEL \cite{Bradbury2007} software version 4.0 accessed from \url{maizegenetics.net/tassel}.  Analyzes utilizing this software's general linear model and mixed linear model functions were conducted using our estimation of CentC abundance as the phenotype.  The general linear model analysis (GLM) relies on a least squares fixed effect linear model to determine association.  The mixed linear model takes into account random effects in addition to the fixed effects included in the GLM, yielding an analysis with a higher statistical power compared to the GLM analysis \cite{Yu2006}.  Q matrices generated by \emph{structure} were employed in both the GLM and MLM analyzes, and omitted in a separate MLM analysis in order to compare the effects of Q + K with K.  Kinship matrices for Jiao et al. dataset \cite{Jiao2012} were generated using TASSEL and included in the MLM analyzes only.    
% citation for glm
% citation for Kinship methods
% clarify Q + K with K comparison
GWAS results were visualized by constructing manhattan and Q-Q plots using the qqman \cite{Turner2014} R package accessed from \url{http://cran.r-project.org/package=qqman}.   

\subsection{Imputation and Identity by Descent} %Tyler


\subsection{Diversity analyses} %Tim


\section{Results}

The natural log probability of the data and  $\Delta$K values for 3 trials were examined, revealing candidate \emph{K} values at 3 and 12 assumed populations.    

\section{Discussion}

\subsubsection*{Acknowledgments}
We would like to thank Anne Lorant for technical assistance, and R. Kelly Dawe, Sofiane Mezmouk, blah, and blah for comments on earlier versions of this manuscript.  This work was funded in part by grant IOS-0922703 from the National Science Foundation.

%----------------------------------------
% REFERENCES
%----------------------------------------
\clearpage
\bibliography{references}

\end{document}
